backend/src/packages/auth/auth.service.ts

```diff
@@ -1,25 +1,53 @@
 import {
+  ApplicationError,
+  ApplicationErrorMessage,
+} from 'shared/build/index.js';
```

Comment:
Do not use direct import from `shared`, these should be imported from enums I believe